src/app/api/routineApi.ts

```diff
@@ -56,3 +56,52 @@ export const fetchRoutines = async (query = ''): Promise<Routine[]> => {
 
   return filteredRoutines;
 };
+
+export const fetchRoutineDetails = async (routineId: string): Promise<Routine> => {
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+
+  const routineDetails: Routine | undefined = routines.find((routine) =>
+    routine.id === routineId
+  );
+
+  if (routineDetails === undefined) {
+    throw new Error();
+  }
+
+  return routineDetails;
+};
+
+export const createRoutine = async (data: {name: string, description?: string}): Promise<Routine> => {
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+
+  const newRoutine: Routine = {
+    id: (Math.random()*1000).toString(),
+    name: data.name,
+    description: data.description,
+    likes: Math.floor(Math.random()*100000),
+    userLikes: false
+  };
+  routines.push(newRoutine);
+
```

Comment:
I understand that this is some intermediate solution. But I think we already can fetch and modify routines from the backend and database.
We already have all the necessary endpoints + we may find bugs when integrating frontend and backend

Reply:
yeah but as mentioned we already have this task in jira in to do